app/src/main/res/layout/local_playlist_header.xml

```diff
@@ -50,7 +50,7 @@
         android:layout_height="wrap_content"
         android:layout_below="@id/playlist_stream_count">
 
-        <include layout="@layout/playlist_control"/>
+        <include layout="@layout/local_playlist_control" />
```

Comment:
Since the local playlist controls have been reset to the old ones, you should revert this change and remove the file `app/src/main/res/layout/local_playlist_control.xml`

Reply:
I should've caught that. 
Thanks for pointing it out.
I have fixed it in commit [fb193ce](https://github.com/TeamNewPipe/NewPipe/pull/3065/commits/fb193ce3622f34b212bff6a645887f98940eb63c)